packages/devp2p/test/integration/eth-simulator.ts

```diff
@@ -208,3 +208,9 @@ test('ETH: invalid status send', async (t) => {
   }
   util.twoPeerMsgExchange(t, opts, capabilities)
 })
+
+test('RLPX: verify that snappy compression does not work with an RLPX peer that only supports devp2p 4', (t) => {
```

Comment:
Sorry, a bit picky here, but can you reword "does not work" -> "is not used", if find this otherwise a bit misleading on a first read.

Reply:
```suggestion
test('RLPX: verify that snappy compression is not used with an RLPX peer that only supports devp2p 4', (t) => {
```